### PR TITLE
Implement nord color scheme

### PR DESCRIPTION
**Pull request by marcusramberg**
Based on https://www.nordtheme.com

### CODE REVIEW

File: assets/sass/_themes.scss [hunk 1]
```diff
@@ -10,8 +10,9 @@ $sidebar-item-icon-width: 1.5rem;
 
 @import 'themes/dark';
 @import 'themes/light';
+@import 'themes/nord';
```
Comment (jhthorsen) [inline assets/sass/_themes.scss]: There's a missing part  in my [post](https://convos.by/2019/11/2/custom-styling.html) about the order: When it's a custom theme, the theme should be before "themes/light". Please re-order it.

**Comment by marcusramberg** [comment]
EId4B18WsAM79Tv
Will fix reordering and get rid of the asset changes.

**Comment by jhthorsen** [comment]
Looks good! Could this be the default dark mode?

**Comment by marcusramberg** [comment]
I think I would prefer for the green one to remain the default, for branding, but I prefer this one myself :)

**Comment by jhthorsen** [comment]
Maybe it’s time to rebrand 😉

**Pull request merged**